HW7/AndriiOliinyk/07_3_Practical_tasks _GitHub_topic_7.py

```diff
@@ -0,0 +1,22 @@
+def count_characters(s):
```

Comment:
good job
look at this version of code
def number_of_characters(text):
    count = {}
    for char in text:
        if char in count:
            count[char] += 1
        else:
            count[char] = 1
    return count